boards/nxp/mimxrt1180_evk/mimxrt1180_evk.dtsi

```diff
@@ -55,20 +55,51 @@
 	status = "okay";
 };
 
-&flexspi1 {
+&flexspi {
+	pinctrl-0 = <&pinmux_flexspi1>;
+	pinctrl-names = "default";
+};
+
+&flexspi {
 	status = "okay";
 	ahb-prefetch;
 	ahb-read-addr-opt;
 	rx-clock-source = <1>;
-	w25q128jw: w25q128jw@0 {
+	w25q128jw: w25q128jw@2 {
 		compatible = "nxp,imx-flexspi-nor";
-		size = <134217728>;
-		reg = <0>;
+		size = <DT_SIZE_M(16*8)>;
+		reg = <2>;
```

Comment:
Reg address follows the port the flash device is on. Address 2 indicates FLEXSPI port B1. Looking at the RT1180 schematic, this should actually be on port A1, right? So the reg address should be 0

Reply:
@lucien-nxp  ping- I believe this needs to be changed.

Reply:
Line above still needs to be updated: `w25q128jw: w25q128jw@2`-> `w25q128jw: w25q128jw@0`